_posts/0000-01-02-lifalakane.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
I have successfully applied changes in to my file as a push request and I'm excited!!!